iam.tf

```diff
@@ -1,9 +1,5 @@
-locals {
-  irsa_role_create = var.enabled && var.rbac_create && var.service_account_create && var.irsa_role_create
-}
-
 data "aws_iam_policy_document" "this" {
```

Comment:
I was doing some testing in LARA and we need to add a local variable `irsa_policy_enabled`:

```
locals {
  irsa_policy_enabled = var.irsa_policy_enabled != null ? var.irsa_policy_enabled : var.irsa_assume_role_enabled != true
}

data "aws_iam_policy_document" "this" {
  count = var.enabled && local.irsa_policy_enabled != false && var.irsa_policy == null ? 1 : 0
```